pysph/examples/surface_tension/equilibrium_rod.py

```diff
@@ -0,0 +1,170 @@
+import numpy as np
+import os
+
+from pysph.sph.surface_tension import return_equations
```

Comment:
As noted below, please change the name of this.

Reply:
Done. Please check.